### PR TITLE
initial version of authentication section

### PR DESCRIPTION
**Pull request by edewit**
## Motivation

updated the admin client to support flows and excutions and implemented front sreeen [marvel](https://marvelapp.com/prototype/bh91013/screen/75647039)

## Brief Description

First page with "delete" and "duplicate" functionality

## Checklist:

- [x] Code has been tested locally by PR requester
- [ ] User-visible strings are using the react-i18next framework (useTranslation)
- [ ] Help has been implemented
- [ ] axe report has been run and resulting a11y issues have been resolved
- [ ] Unit tests have been created/updated
- [x] Formatting has been performed via prettier/eslint
- [x] Type checking has been performed via 'yarn check-types'

## Additional Notes

image

### CODE REVIEW

File: src/authentication/messages.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "authentication": {
+    "title": "Authentication",
+    "flows": "Flows",
+    "flowName": "Flow name",
+    "usedBy": "Used by",
+    "buildIn": "Build in",
```
Comment (ssilvert) [inline src/authentication/messages.json:7]:
```suggestion
    "buildIn": "Built-in",
```

Maybe should change the key as well.  The English expression is "Built-in"

File: src/authentication/messages.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "appliedByProviders": "Applied by the following providers",
+    "appliedByClients": "Applied by the following clients",
+    "specificProviders": "Specific providers",
+    "specificClients": "Specific clients",
+    "default": "Default",
+    "notInUse": "Not in use",
+    "duplicate": "Duplicate",
+    "deleteConfirmFlow": "Delete flow?",
+    "deleteConfirmFlowMessage": "Are you sure you want to permanently delete the flow \"<1>{{flow}}</1>\".",
+    "deleteFlowSuccess": "Flow successfully delete",
```
Comment (ssilvert) [inline src/authentication/messages.json:17]:
```suggestion
    "deleteFlowSuccess": "Flow successfully deleted",
```

File: src/authentication/AuthenticationSection.tsx [hunk 3]
```diff
@@ -1,6 +1,250 @@
...
+        <KeycloakTabs isBox>
+          <Tab
+            eventKey="flows"
+            title={<TabTitleText>{t("flows")}</TabTitleText>}
+          >
+            <KeycloakDataTable
+              key={key}
+              loader={loader}
+              ariaLabelKey="authentication:title"
+              searchPlaceholderKey="authentication:searchForEvent"
```
Comment (ssilvert) [inline src/authentication/AuthenticationSection.tsx:195]: Should be `searchForFlow`.  Needs key/value pair as well.

**Review by ssilvert — CHANGES_REQUESTED** [review]
Table does not update after doing Duplicate flow.

On an unrelated note, REST API lets you duplicate a new flow to the same Flow name.  Old console has same issue.

File: src/authentication/AuthenticationSection.tsx (hunk 3 above)
Comment (edewit) [reply src/authentication/AuthenticationSection.tsx:195]: This screen doesn't have a search input text, but the component can't handle that yet

**Comment by ssilvert** [comment]
@edewit Looked at this again.  It's fine except that the table doesn't update when you duplicate.  Do you want to fix that now or open an issue?

**Review by ssilvert — APPROVED** [review]

**Pull request merged**